integration/network/network_test.go

```diff
@@ -258,3 +258,21 @@ func TestDefaultNetworkOpts(t *testing.T) {
 		})
 	}
 }
+
+func TestForbidDuplicateNetworkNames(t *testing.T) {
+	skip.If(t, testEnv.DaemonInfo.OSType == "windows")
```

Comment:
Do you know why this didn't fail on Windows?

Reply:
I force pushed my commit to check what's the error, and here it is:

```
=== Failed
=== FAIL: github.com/docker/docker/integration/network TestForbidDuplicateNetworkNames (0.01s)
    network_test.go:266: [d46d27c88db30] failed to start daemon with arguments [--data-root D:\a\moby\moby\go\src\github.com\docker\docker\bundles\tmp\TestForbidDuplicateNetworkNames\d46d27c88db30\root --exec-root C:\Users\RUNNER~1\AppData\Local\Temp\dxr\d46d27c88db30 --pidfile D:\a\moby\moby\go\src\github.com\docker\docker\bundles\tmp\TestForbidDuplicateNetworkNames\d46d27c88db30\docker.pid --userland-proxy=true --containerd-namespace d46d27c88db30 --containerd-plugins-namespace d46d27c88db30p --containerd /var/run/docker/containerd/containerd.sock --host unix://C:\Users\RUNNER~1\AppData\Local\Temp\docker-integration\d46d27c88db30.sock --debug] : protocol not available
```

So, it's totally unrelated to this change. I'll create a follow-up PR addressing that.

Reply:
Interesting; thanks!

(mostly commented because we have many tests with a `skip`, and for some nobody even remembers "why" 😂)